src/scripts/main.js

```diff
@@ -1,7 +1,39 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const createNotificationElement = () => {
+    const notification = document.createElement('div');
+
+    notification.className = `notification ${type}`;
+    notification.style.position = 'fixed';
+    notification.style.top = `${posTop}px`;
+    notification.style.right = `${posRight}px`;
+
+    const titleElement = document.createElement('h2');
+
+    titleElement.className = 'title';
+    titleElement.textContent = title;
+    notification.appendChild(titleElement);
+
+    const descriptionElement = document.createElement('p');
+
+    descriptionElement.textContent = description;
+    notification.appendChild(descriptionElement);
+
+    return notification;
+  };
+
+  const showNotification = (element) => {
+    document.body.appendChild(element);
+
+    setTimeout(() => {
+      document.body.removeChild(element);
+    }, 2000);
+  };
+
+  const notificationElement = createNotificationElement();
+
+  showNotification(notificationElement);
 };
```

Comment:
The function `pushNotification` does a lot of things: it creates a notification element, shows it, and removes it. It would be better to have separate functions for each of these operations. This would make your code more modular and easier to understand. You could have a function `createNotification` that creates and returns a notification element, a function `showNotification` that takes a notification element and shows it, and a function `removeNotification` that removes a notification after a certain period of time.